### PR TITLE
ior: ime variant

### PR DESCRIPTION
**Pull request by heerener**

### CODE REVIEW

File: var/spack/repos/builtin/packages/ior/package.py [hunk 1]
```diff
@@ -64,4 +65,9 @@ def configure_args(self):
         else:
             config_args.append("--without-ncmpi")
 
+        if "+ime" in self.spec:
+            config_args.append("--with-ime")
+        else:
+            config_args.append("--without-ime")
+
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/ior/package.py]: You could use `config_args.append(self.with_or_without("time"))` here (see https://spack.readthedocs.io/en/latest/build_systems/autotoolspackage.html#with-or-without).

**Review by tldahlgren — DISMISSED** [review]
LGTM though I *recommend* using `self.with_or_without()`.  I realize it isn't being used above.

Let me know if you want the PR merged as-is or want to make this change.

File: var/spack/repos/builtin/packages/ior/package.py (hunk 1 above)
Comment (heerener) [reply var/spack/repos/builtin/packages/ior/package.py]: I gave it a try and everything still builds - thanks for pointing this out!

**Review by bernhardkaindl — COMMENTED** [review]
@heerener: 

Building `bin/spack install ior+ime` fails with many files missing ime_native.h for me:

```py
$ make
Making all in src
make  all-recursive
Making all in .
  CC       ior-aiori-IME.o
aiori-IME.c:31:10: fatal error: ime_native.h: No such file or directory
   31 | #include "ime_native.h"
      |          ^~~~~~~~~~~~~~
```

https://github.com/DDNStorage/ime_native/blob/master/ime_native.h

Is it  this file you are using?

If so, you must add it, maybe copy it add it as an additional source file.

**Comment by alalazo** [comment]
Closing, since there has been no activity in a while. Feel free to reopen if you want to continue work on this.

**Pull request closed without merging**